docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### storage_page_cache_limit
+
+- **Default:** 20%
+- **Description:** The PageCache size. It can be specified as size, for example, `20G`, `20,480M`, `20,971,520K`, or `21,474,836,480B`. It can also be specified as the ratio (percentage) to the memory size, for example, `20%`. It takes effect only when `disable_storage_page_cache` is set to `false`.
+
+#### internal_service_async_thread_num
+
+- **Default:** 10 (Number of Threads)
+- **Description:** The thread pool size allowed on each BE for interacting with Kafka. Currently, the FE responsible for processing Routine Load requests depends on BEs to interact with Kafka, and each BE in StarRocks has its own thread pool for interactions with Kafka. If a large number of Routine Load tasks are distributed to a BE, the BE's thread pool for interactions with Kafka may be too busy to process all tasks in a timely manner. In this situation, you can adjust the value of this parameter to suit your needs.
+
+### Configure BE static parameters
+
+You can only set the static parameters of a BE by changing them in the corresponding configuration file **be.conf**, and restart the BE to allow the changes to take effect.
+
+BE static parameters are as follows.
+
+#### hdfs_client_enable_hedged_read
+
+- **Default**: false
+- **Unit**: N/A
+- **Description**: Specifies whether to enable the hedged read feature. This parameter is supported from v3.0 onwards.
+
+#### hdfs_client_hedged_read_threadpool_size
+
+- **Default**: 128
+- **Unit**: N/A
+- **Description**: Specifies the size of the Hedged Read thread pool on your HDFS client. The thread pool size limits the number of threads to dedicate to the running of hedged reads in your HDFS client. This parameter is supported from v3.0 onwards. It is equivalent to the dfs.client.hedged.read.threadpool.size parameter in the hdfs-site.xml file of your HDFS cluster.
+
+#### hdfs_client_hedged_read_threshold_millis
+
+- **Default**: 2500
+- **Unit**: Millisecond
+- **Description**: Specifies the number of milliseconds to wait before starting up a hedged read. For example, you have set this parameter to 30. In this situation, if a read from a block has not returned within 30 milliseconds, your HDFS client immediately starts up a new read against a different block replica. This parameter is supported from v3.0 onwards. It is equivalent to the dfs.client.hedged.read.threshold.millis parameter in the hdfs-site.xml file of your HDFS cluster.
+
+#### be_port
+
+- **Default**: 9060
+- **Unit**: N/A
+- **Description**: The BE thrift server port, which is used to receive requests from FEs.
+
+#### brpc_port
+
+- **Default**: 8060
+- **Unit**: N/A
+- **Description**: The BE BRPC port, which is used to view the network statistics of BRPCs.
+
+#### brpc_num_threads
+
+- **Default**: -1
+- **Unit**: N/A
+- **Description**: The number of bthreads of a BRPC. The value -1 indicates the same number with the CPU threads.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/67ec571f-563f-4a4b-8832-cf166d7dc0d3)
Warp "-1" in code format